src/styles/helpers/__test__/clamp.spec.js

```diff
@@ -0,0 +1,24 @@
+import clamp from '../color/clamp';
+
+describe('clamp', () => {
+    it('should return the same value if it is between 0 and 1', () => {
+        expect.assertions(4);
+        [0, 0.1, 0.5, 1].forEach(v => {
+            expect(clamp(v)).toStrictEqual(v);
```

Comment:
For assertions against primitive values (strings, number, and booleans) use toBe.
toStrictEqual is used with objects:
https://jestjs.io/docs/en/expect#tostrictequalvalue